File: 1020.cs
using System;
class URI {
    static void Main(string[] args) { 
        int n1 = int.Parse(Console.ReadLine());
        int ano = n1 / 365;
        n1 = n1 - ano * 365;
        int mes = n1 / 30;
        n1 = (n1 - mes * 30);
        Console.WriteLine($"{ano} ano(s)\n{mes} mes(es)\n{n1} dia(s)");
    }
}

File: 1020.cpp
#include <iostream>
using namespace std;

int main(){
    int n1, ano, mes;
    cin >> n1;
    ano = n1 / 365;
    n1 = n1 - ano * 365;
    mes = n1 / 30;
    n1 = (n1 - mes * 30);
    cout << ano << " ano(s)" << "\n";
    cout << mes << " mes(es)" << "\n";
    cout << n1 << " dia(s)" << endl;
}
